AmazonChimeSDK/AmazonChimeSDK/audiovideo/video/VideoCodecCapability.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  VideoCodecCapability.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+@objcMembers public class VideoCodecCapability: NSObject {
+    public var name: String = ""
+    public var clockRate: Int32 = 0
+    public var parameters: [String: String] = [:]
+
+    public init(name: String, clockRate: Int32, params: [String: String]) {
+        self.name = name
+        self.clockRate = clockRate
+        self.parameters = params
+    }
+
+    public static func vp8() -> VideoCodecCapability {
+        return VideoCodecCapability(name: "VP8", clockRate: 90000, params: [:])
+    }
+
+    public static func h264cbp() -> VideoCodecCapability {
```

Comment:
nit: spell out `ConstrainedBaselineProfile`. 'CBP' is not necessarily a common shortening.